lib/logger.ts

```diff
@@ -54,6 +54,8 @@ export class Logger {
   static set(config: Config): void {
     if (config.troubleshoot) {
       Logger.logLevel = LogLevel.DEBUG;
+    } else if (config.logLevel) {
+      Logger.logLevel = (<any>LogLevel)[config.logLevel];
```

Comment:
What is `(<any>LogLevel)` accomplishing here? I'm able to compile without it (that said, I'm not a typescript expert, so perhaps I am missing something).